.github/workflows/docker.yml

```diff
@@ -34,3 +34,5 @@ jobs:
           platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tags }}
           push: true
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
Can you explain me this please?

Reply:
This is to tell the Docker build step to push/fetch data from the Github Cache so building the container is faster on subsequent runs.

Reply:
Thanks